packages/blade/src/components/Input/_decisions/decisions.md

```diff
@@ -0,0 +1,295 @@
+| onClearButtonClicked                                                                               | `(event) => {}`                                                           | No       |                                                           | Used in conjunction with `showClearButton` prop. Provides the capability to the user to handle/control the behavior of clear icon button. Can be used to send analytics event and can also be used to clear the content when the input field is `controlled`                          |
+| prefix                                                                                             | `string`                                                                  | No       |                                                           | Used when we want to add prefix to the input field. Eg: `$` or `₹`. The prefix will be placed **after** `icon`                                                                                                                                                                        |
+| suffix                                                                                             | `string`                                                                  | No       |                                                           | Used when we want to add suffix to the input field. Eg: `.00` or `@gmail.com`. If `showClearButton` is `true` then suffix will be placed **after** clear button.                                                                                                                      |
+| isLoading                                                                                          | `boolean`                                                                 | No       | `false`                                                   | Used to show a spinner when the the data is being loaded. **Discuss the position of the loader. Check options mentioned in the open questions below**                                                                                                                                 |
+| maxCharacters                                                                                      | `number`                                                                  | No       |                                                           | Used to show a character counter under the input field                                                                                                                                                                                                                                |
```

Comment:
**Question:** Same question as for `isDisabled`, possible to have an alias which is the same as the original API maxlength?